apps/web/public/static/locales/en/common.json

```diff
@@ -59,8 +59,8 @@
   "integration_meeting_id": "{{integrationName}} meeting ID: {{meetingId}}",
   "confirmed_event_type_subject": "Confirmed: {{eventType}} with {{name}} at {{date}}",
   "new_event_request": "New event request: {{attendeeName}} - {{date}} - {{eventType}}",
-  "confirm_or_reject_request": "Confirm or reject the request",
-  "check_bookings_page_to_confirm_or_reject": "Check your bookings page to confirm or reject the booking.",
+  "confirm_or_reject_request": "Confirm or decline the request",
```

Comment:
If we're going for the usage of the word 'decline', should we change 'confirm' to 'accept'?

Reply:
interesting